chrome/content/composeOverlay.js

```diff
@@ -25,11 +25,33 @@ var ReplyAsOriginalRecipient = {
     if (!this.isReply())
       return;
 
-    /* Get original recipient */
-    originalHeader = this.getMessageHeaderFromURI(gMsgCompose.originalMsgURI);
-    originalRecipient = originalHeader.mime2DecodedRecipients;
-    if (originalRecipient.indexOf(",") != -1 || originalRecipient.indexOf("+") == -1)
-      return;
+    /* Get patterns preference (modified by Samuel Kirschner, according to the comment on https://blog.qiqitori.com/?p=194 ) */
+    var patterns = Components.classes["@mozilla.org/preferences-service;1"].getService(Components.interfaces.nsIPrefService).getBranch("extensions.replyasoriginalrecipient.").getCharPref("patterns"); // default is "*+*"
+    var mimeConvert = Components.classes["@mozilla.org/messenger/mimeconverter;1"]
+        .getService(Components.interfaces.nsIMimeConverter);
+	patterns = patterns.trim().replace(/[^a-zA-Z0-9 ]/g, '\\$&').replace(/\\\*/g, '.*').split(/ *\\, */).join('|');
```

Comment:
Hi, thank you for your patch. Can you add a comment for this line (and maybe break down the replaces/splits/joins into separate lines)? There are also some indentation issues that would be nicer to get fixed, but on the whole these commits are looking good.

Reply:
I did as you requested.